# Bioinfo-Resources
A Collection of useful bioinfo resources, including webtools, databases, packages, tutorials, etc

# Single-cell tutorials
1. Single-cell best practices book (Python, scanpy): https://www.sc-best-practices.org/preamble.html


# Online analysis Platforms
1. Parse Biosciences: https://app.trailmaker.parsebiosciences.com/
2. Flux estimator: http://scflux.org/
3. Cell Marker 2.0: http://117.50.127.228/CellMarker/CellMarker_annotation.jsp
4. 

# R packages
1. GeneNMF: unsupervised discovery of gene programs in single-cell data
2. scRNAtoolVis: Visualization utilities for single-cell data
3. ShinyCell: Generate shiny app for annotated single-cell
4. hdWCGNA: high-dimentional WCGNA for single-cell data
5. CellRank 2: Unified fate mapping in multiview single-cell data
6. scFEA: A graph neural network model to estimate cell-wise metabolic using single cell RNA-seq data
7. InferCNV: inferring copy number variation in single-cell data
8. SCEVAN: inferring copy number variation in single-cell data, categorizing cancer/normal cells
9. CopyKat: inferring copy number variation in single-cell data
10. Cellchat: inferring cell-cell communication
11. scType: Automatic cell annotation using markers
12. DoubletFinder: find potential doublets/multiplets in single-cell data
13. ALRA: Adaptively-thresholded Low Rank Approximation (ALRA)
